### DISCUSSION TITLE
Skip - init does NOT generate a GIT local machine repo

### DISCUSSION THREAD

**Discussion by davidakoontz** [skiptools/skip#225] [Ideas]
I don't know the reasoning - but seems the SKIP INIT for using Fastlane command should start with a REPO.  The whole purpose of Fastlane is to move all the CI/CD configuration stuff into a controllable REPO... the command:

**skip init --fastlane --appid=com.goodcompany.HelloWorld helloworld HelloWorld**

Should start out with a local machine REPO.

**Comment by marcprux** [comment]
The source control management system that a project uses is outside the scope of Skip's project generation. Even though git is very common, it is not the only SCM that developers and organizations use.

It is simple enough to just run `git init` after you create a project.
